docs/prow/authorization.md

```diff
@@ -6,11 +6,12 @@ To deploy a Prow cluster, configure the following service accounts in the GCP pr
 
 | Service account name          | Usage                                                      | Required roles |
 | :---------------------------- | :----------------------------------------------------------| :------------- |
-| **sa-gcs-plank**              | Used by Prow plan microservice | `Storage Object Admin`
-| **sa-gke-kyma-integration**   | Running integration tests on GKE cluster | `Compute Admin`, `Kubernetes Engine Admin`, `Kubernetes Engine Cluster Admin`, `DNS Administrator`, `Service Account User`, `Storage Admin`
-| **sa-kyma-artifacts**         | Saving release artifacts to the GCS bucket | `Storage Object Admin`
-| **sa-vm-kyma-integration**    | Running integration tests on minikube | `Compute Instance Admin (beta)`, `Compute OS Admin Login`, `Service Account User`
-| **sa-gcr-push-kyma-project**  | Publishing docker images | `Storage Admin`
+| **sa-gcs-plank**              | Used by Prow plan microservice | `Storage Object Admin` (`roles/storage.objectAdmin`) in the `kyma-prow-logs` bucket
+| **sa-gke-kyma-integration**   | Running integration tests on GKE cluster | `Compute Admin` (`roles/compute.admin`), `Kubernetes Engine Admin` (`roles/container.admin`), `Kubernetes Engine Cluster Admin` (`roles/container.clusterAdmin`), `DNS Administrator` (`roles/dns.admin`), `Service Account User` (`roles/iam.serviceAccountUser`), `Storage Admin` (`roles/storage.admin`)
+| **sa-kyma-artifacts**         | Saving release artifacts to the GCS bucket | `Storage Object Admin` (`roles/storage.objectAdmin`)
```

Comment:
not only release artifacts but also development artifacts

Reply:
done